database/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM postgres:9.6.2-alpine
+
+RUN apk update \
+&& apk upgrade \
+&& apk add -q alpine-sdk \
+&& apk add -q vim \
+&& apk add -q nano;
+
```

Comment:
I just want to make sure that if we open a shell inside the Docker container for the database, we can query it with psql.